cmd/containerd-shim-runhcs-v1/service_internal.go

```diff
@@ -454,3 +455,17 @@ func (s *service) shutdownInternal(ctx context.Context, req *task.ShutdownReques
 	os.Exit(0)
 	return empty, nil
 }
+
+func (s *service) getComputeProcessorInfo(ctx context.Context, req *extendedtask.ComputeProcessorInfoRequest) (*extendedtask.ComputeProcessorInfoResponse, error) {
```

Comment:
Rename this `computeProcessInfoInternal` to match naming scheme.

Reply:
Done